pyvista/plotting/axes.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, show_actor=False, actor_scale=1, line_width=1.0, symmetric=Fa
         self.mapper.SetInputConnection(self.GetOutputPort())
         # Add the axes actor
         self.actor = Actor(mapper=self.actor)
+        self.axes_actor = _vtk.vtkAxesActor()
```

Comment:
Although this works, why having two actors within the same class, see `self.actor` and `self.axes_actor`.

Ideally, it should be possible to do:

```python
axes = pv.Axes()
axes.actor.GetXAxisShaftProperty().SetColor(1, 0, 0)
```

instead of what the current implementation provides:

```python
axes = pv.Axes()
axes.axes_actor.GetXAxisShaftProperty().SetColor(1, 0, 0)

Reply:
In VTK, `vtkActor` and `vtkAxesActor` are two different things. If you check on [VTK documentation for `vtkAxesActor`](https://vtk.org/doc/nightly/html/classvtkAxesActor.html), you can see that `vtkActor` is just a dependency of `vtkAxesActor`, not a parent class as you would expect. 

So, despite being able to make `Actor` class inherit from `vtkAxesActor`, I don't think that it is correct, since they are two different and somewhat unrelated classes. It would cause that if you build an `Actor` object, unrelated to the axes, that object could access the axes properties, which is prone to errors when developing.

This is the reason I decided to set two `Actor` attributes. To avoid this, I could create a wrapper for `vtkAxesActor`, which I think that could work to substitute both `Actor` attributes. Anyway, let me know what do you think 😄

Reply:
Thanks for the thorough investigation, @AlejandroFernandezLuces! I'm curious why this object isn't used anywhere else though?

Reply:
Thanks for the deep explanation this, @AlejandroFernandezLuces.

Reply:
`vtkAxesActor` is wrapped in the following function, which is used for processing the [parameters in `Plotter.add_axes()`](https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.add_axes.html#add-axes). I suppose this customization is not what @jorgepiloto is looking for.


https://github.com/pyvista/pyvista/blob/25a482b4d1abc6c8472b962ba657cc03a376e707/pyvista/plotting/tools.py#L106-L244